vue/components/forms/personalization-form/initials-images.vue

```diff
@@ -215,6 +215,14 @@ export const InitialsImages = {
             if (this.imageBorderRadius) base["border-radius"] = `${this.imageBorderRadius}`;
             if (this.imageObjectFit) base["object-fit"] = this.imageObjectFit;
             return base;
+        },
+        _openExternally() {
+            if (this.openExternally !== null) return this.openExternally;
+
+            const features = this.$store.state.features;
+            if (features["open-externally"] !== undefined) return features["open-externally"];
+
+            return false;

```

Comment:
can't you simplify with something like
```suggestion
            const features = this.$store.state.features || {};
            return features["open-externally"] !== undefined ? features["open-externally"] : this.openExternally; 
```

Reply:
In this case, what if neither the feature nor the prop is defined? You get null instead of false.

Reply:
The boolean value of null is false, so it's not a problem. Either way, why change the default value of the prop to null instead of false?

Reply:
Because if it is false, I can not know if the prop was given or defaulted, and hence I cant use the logic of first the prop, then the feature, then false default.

Reply:
I forgot you wanted to give priority to the prop, perhaps something like:
```suggestion
            const features = this.$store.state.features || {};
            if (this.openExternally !== null) return this.openExternally;
            return features["open-externally"] !== undefined ? features["open-externally"] : this.openExternally; 
```

Reply:
This would return null if no prop or feature is given. I know in javascript it will have the same behavior, but I really think we should return false for a boolean property.

Reply:
Sure